atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -257,7 +257,6 @@ public String toString() {
 
         // immediately throw when we try and borrow from a full pool; dealt with at higher level
         poolConfig.setBlockWhenExhausted(false);
-        poolConfig.setMaxWaitMillis(config.socketTimeoutMillis());
```

Comment:
`MaxWaitMillis` is ignored when `BlockWhenExhausted` is false.
This configuration means how long to block if the pool is exhausted.
It is also weird that this was getting set to the socket timeout millis. This seemed like an oversight

Reply:
Note that this was originally added in a PR when `BlockWhenExhausted` was true: https://github.com/palantir/atlasdb/commit/99de2f76d2ea9b47ba16b3eb0c3ffc52970d3278#diff-b88d7ebe9dabc333cf7e4dff942e1830R149

Reply:
Yep, confirming that this is correct - it is indeed ignored (from `GenericObjectPool::borrowObject`)